client/src/store/stores/interaction.ts

```diff
@@ -144,4 +191,13 @@ export default class InteractionStore {
 
     return index;
   }
+
+  private static convertHubbleFlow(hubbleFlow: HubbleFlow): Flow {
```

Comment:
I feel this is wrong place to provide domain types conversion method
We have `~/domain/helpers` doing the same thing